index.js

```diff
@@ -161,8 +161,10 @@ export const Classes = {
 	activities: {
 		assigned: 'assigned',
 		complete: 'complete',
+		completion: 'completion',
 		exempt: 'exempt',
 		feedbackDate: 'feedback-date',
+		relativeUri: 'relative-uri',
```

Comment:
This one seems like it could be in the root domain, since it's not just an activities thing - we actually use relative URIs in other places, might be good to add this class to them. (Not saying you should do this, this is more of a self-note whenever I come across one)

Reply:
"root domain" hah I thought this was a rel, not a class. Either way, `Classes.relativeUri` rather than `Classes.activities.relativeUri` is that I meant :smile:

Reply:
I'd agree with this point. If this is a thing we are going to be supporting, preferably in the same way in both Homepages and now in activities ( until we can fix the _actual_ issue of cookies and whatnot ), it should be a core concept, that both those domains share.